### PR TITLE
#10412

### LINKED ISSUE #10412: The documentation page for the SeDelegateSessionUserImpersonatePrivilege privilege is missing
The group policy setting *Obtain an impersonation token for another user in the same session* links to this page (https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/impersonate-a-client-after-authentication) which corresponds to another setting, *Impersonate a client after authentication*. Is this the correct behavior?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7de1fb45-91eb-f37f-8106-81bf82a5602e
* Version Independent ID: 1c765989-4c50-4b89-83fe-a8625f0f9fd5
* Content: [User Rights Assignment (Windows 10) - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/user-rights-assignment)
* Content Source: [windows/security/threat-protection/security-policy-settings/user-rights-assignment.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-policy-settings/user-rights-assignment.md)
* Product: **m365-security**
* Technology: **windows-sec**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by michaelAngeloEgypt**
fixes #10412

### CODE REVIEW

File: windows/security/threat-protection/security-policy-settings/user-rights-assignment.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ The following table links to each security policy setting and provides the const
 | [Manage auditing and security log](manage-auditing-and-security-log.md)| SeSecurityPrivilege| 
 | [Modify an object label](modify-an-object-label.md) | SeRelabelPrivilege| 
 | [Modify firmware environment values](modify-firmware-environment-values.md)| SeSystemEnvironmentPrivilege| 
-| [Obtain an impersonation token for another user in the same session](impersonate-a-client-after-authentication.md) | SeDelegateSessionUserImpersonatePrivilege|
+| [Obtain an impersonation token for another user in the same session](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/windows-server/windows-security/seImpersonateprivilege-secreateglobalprivilege.md) | SeDelegateSessionUserImpersonatePrivilege|
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/user-rights-assignment.md:74]:
```suggestion
| [Obtain an impersonation token for another user in the same session](/troubleshoot/windows-server/windows-security/seimpersonateprivilege-secreateglobalprivilege) | SeDelegateSessionUserImpersonatePrivilege|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @michaelAngeloEgypt, kindly commit the suggested change. Many thanks.

**Comment by aczechowski** [comment]
Hold off...starting a discussion on original issue

**Comment by e0i** [comment]
Link to discussion at the original issue, https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10412#issuecomment-1182636666

**Comment by aczechowski** [comment]
@michaelAngeloEgypt this PR doesn't resolve the associated issue. I think we need a new article for the SeDelegateSessionUserImpersonatePrivilege user right. My team can use that issue to track that effort. I'm closing this PR - feel free to reopen if you feel differently or I'm missing something.

**Pull request closed without merging**